msal/src/main/java/com/microsoft/identity/client/PublicClientApplication.java

```diff
@@ -43,6 +43,7 @@
 import com.microsoft.identity.client.internal.MsalUtils;
 import com.microsoft.identity.client.internal.configuration.LogLevelDeserializer;
 import com.microsoft.identity.client.internal.controllers.BrokerMsalController;
+import com.microsoft.identity.client.internal.controllers.LocalMSALController;
 import com.microsoft.identity.client.internal.controllers.MSALControllerFactory;
```

Comment:
Not part of your change but since MSAL is an acronym, would be good to change all MsalController to MSALController.

Reply:
As best we can, we try to adhere to the [Android Style Guide](https://source.android.com/setup/contribute/code-style#treat-acronyms-as-words) which advocates treating acronyms as words. As we have a lot of contributors there's been some inconsistency on this point; however, we ought to stylize as `MsalClassName` rather than `MSALClassName`